.github/workflows/functional_tests.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v3
         with:
-          submodules: 'true'
+          submodules: 'recursive'
```

Comment:
Should docs\dev-setup\index.md be updated to add --recursive to 'git submodule update --init'?

Reply:
Good catch! added that.